### PR TITLE
[Storage]  GA blob inventory

### PR DESCRIPTION
**Pull request by Juliehzl**
**Description**

GA blob inventory feature, which is in storage-preview extension before.

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Storage

File: src/azure-cli/azure/cli/command_modules/storage/_help.py [hunk 1]
```diff
@@ -22,6 +22,43 @@
...
+type: group
+short-summary: Manage storage account Blob Inventory Policy.
+"""
+
+helps['storage account blob-inventory-policy create'] = """
+type: command
+short-summary: Create Blob Inventory Policy for storage account.
+examples:
+  - name: Create Blob Inventory Policy trough json file for storage account.
+    text: az storage account blob-inventory-policy create -g ResourceGroupName --account-name storageAccountName --policy @policy.json
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/storage/_help.py:35]:
```suggestion
    text: az storage account blob-inventory-policy create -g myresourcegroup --account-name mystorageaccount --policy @policy.json
```
Same below~

**Review by evelyn-ys — APPROVED** [review]
LGTM

**Comment by Juliehzl** [comment]
wait for service team feedback for Preview tag and help link

**Pull request merged**